src/listpack.c

```
@@ -540,9 +547,8 @@ lpGetWithSize(unsigned char *p, int64_t *count, unsigned char *intbuf, uint64_t
         if (entry_size) *entry_size = 5 + *count + lpEncodeBacklen(NULL, *count + 5);
         return p + 5;
     } else {
-        uval = 12345678900000000ULL + p[0];
-        negstart = UINT64_MAX;
-        negmax = 0;
+        serverLog(LL_WARNING, "Invalid listpack encoding. Byte %02hhx is not a valid encoding.", p[0]);
```

I would suggest avoid adding server dependency in this stage so we can keep the listpack as a standalone DS as much as possible. probably with @madolson suggestion (later) to use panicprintf we can have the same data printed.

---

There is no `panicprintf()` function, afaik. I had to add `server.h` for `serverLog()`.

---

Apparently the args of panic is `panic(const char * fmt, ...)`, I thought for some reason it actually didn't take a format. I think we can just do `panic("Invalid listpack encoding. Byte %02hhx is not a valid encoding.", p[0])`.